kratos/tests/geometries/test_triangle_2d_3.cpp

```diff
@@ -57,6 +57,19 @@ namespace Testing {
     ));
   }
 
+  /** Generates a sample triangle2D3.
+   * Generates a right triangle with origin in (1,1).
+   * @return  Pointer to a triangle2D3
+   */
+  template<class TPointType>
```

Comment:
You can generate a random triangle with something @roigcarlo implemented

Reply:
I know, but if I'm not wrong it generates a random 3D triangle.

Reply:
Yep, just in case you didn't know

Reply:
I don't like random tests (except for stress tests) as they randomly crash without giving information unless you write done the used value in failing message.

Reply:
This is the reason why I implemented this new generate triangle function.